api/src/main/java/org/pmiops/workbench/billing/BillingProjectUtil.java

```diff
@@ -0,0 +1,27 @@
+package org.pmiops.workbench.billing;
+
+import com.google.common.hash.Hashing;
+import java.util.UUID;
+import javax.inject.Provider;
+import org.pmiops.workbench.config.WorkbenchConfig;
+
+/** Utilities for Billing Project related methods. */
+public class BillingProjectUtil {
+  private static final int PROJECT_BILLING_ID_SIZE = 8;
+
+  /** Creates a random Billing Project name. */
+  public static String createBillingProjectName(Provider<WorkbenchConfig> workbenchConfigProvider) {
```

Comment:
Why did you decide to pass the workbenchConfigProvider instead of just the projectNamePrefix?

Reply:
that is a good idea, I will switch to your suggestion

Reply:
alternatively, maybe we can make it a new service? That way it can import the config dependency on its own without relying on its callers to pass it in. It's a bit overkill for just this method but I think it will be necessary down the road if this utility class grows in functionality and acquires more dependencies

Reply:
Now I switch to just pass a String prefix without knowing any of the config.  I do feel a Service is little bit overkill for now, and I'd like that convert to a service if that getting complicated.
In the meanwhile, I am exploring see if we can make test setup/injection easier with new Spring test runner